src/internal/_Set.js

```diff
@@ -0,0 +1,167 @@
+var _contains = require('./_contains');
+
+
+// A simple Set type that honours R.equals semantics
+module.exports = (function() {
+  function _Set() {
+    /* globals Set */
+    this._nativeSet = typeof Set === 'undefined' ? null : new Set();
+    this._items = {};
+    this._negZeroExists = false;
+    this._nullExists = false;
+    this._undefExists = false;
+    this._falseExists = false;
+    this._trueExists = false;
```

Comment:
Yeah now that we have an `_items` bucket, we might as well store the others in there too – updated :zap: